pkg/common/common.go

```diff
@@ -145,3 +146,42 @@ func IsContextDone(ctx context.Context) bool {
 		return false
 	}
 }
+
+func GetLocalIPs() ([]string, error) {
```

Comment:
Also account for the possibility that it is DNS names and not actual IP addresses in the cluster member list. So if dns name, resolve the name and check if that IP is plumbed.

Reply:
Yes, `GetIPFromHostName` in this file performs lookup for hostnames.

This is called in controller.go when identifying the local member's socket in order to resolve all the target Member IP addresses.